walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -93,4 +93,16 @@ private static boolean equals(Event a, Event b) {
     // interface documentation, equals will check for set-equality across all set implementations.
     return a.title.equals(b.title) && a.when.equals(b.when) && a.attendees.equals(b.attendees);
   }
+
+  /**
+   * Returns true iff at leas one attendee in requestedAttendees participates in this Event 
+   */
+  public boolean containsRequestedAttendees(Collection<String> requestedAttendees) {
+    for (String requestedAttendee : requestedAttendees) {
+      if (attendees.contains(requestedAttendee)) {
```

Comment:
Tip: This is fine as is so you don't need to change anything, but Java 8 added some nice new features for handling "streams" of data which allows for some more declarative processing of arrays and other "streamlike" things.

Sometimes using the declarative approach makes your code more concise and readable, so it is a a useful thing to have in your toolbox.

Some docs from Oracle: https://www.oracle.com/technical-resources/articles/java/ma14-java-se-8-streams.html

Reply:
That's interesting, I will change the code to use streams after I finish testing.